src/main/scala/com/codacy/duplication/pmd/Cpd.scala

```diff
@@ -135,10 +135,10 @@ object Cpd extends DuplicationTool {
   }
 
   private def duplicationClone(m: Match, rootDirectory: Path): DuplicationClone = {
-    val files: List[DuplicationCloneFile] = m.getMarkSet.asScala.map { mark =>
+    val files: List[DuplicationCloneFile] = m.getMarkSet.asScala.view.map { mark =>
```

Comment:
whats the advantage of having `.view` here? please teach 🙏  🤓

Reply:
is it only a scala 2.13 compatibility thing?

Reply:
Scala `2.13` replaced `collection.breakOut` with views: https://docs.scala-lang.org/overviews/core/collections-migration-213.html

Reply:
🤘